java/vector/src/main/java/org/apache/arrow/vector/BaseDataValueVector.java

```diff
@@ -58,9 +58,7 @@ public static void truncateBufferBasedOnSize(List<ArrowBuf> buffers, int bufferI
     }
     return result;
   }
-
-  // TODO: Nullable vectors extend BaseDataValueVector but do not use the data field
-  // We should fix the inheritance tree
```

Comment:
@siddharthteotia I believe you resolved this comment in #892.  Does this PR look ok to you?